src/style.css

```diff
@@ -1,3 +1,90 @@
+:root {
+  --color-on-hover: #00acdc;
```

Comment:
👍 for using variables, however, I suggest you rename it.
Better names are `accent-color` or even `moyo-blue`. Current name is a bit misleading.